vm/address.go

```diff
@@ -21,5 +21,8 @@ var EndOfEpochAddress = []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 255,
 // DelegationManagerSCAddress is the hard-coded address for the delegation manager smart contract
 var DelegationManagerSCAddress = []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 255, 255}
 
+// LiquidStakingSCAddress is the hard-coded address for the delegation token smart contract
+var LiquidStakingSCAddress = []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5, 255, 255}
```

Comment:
maybe add in the /examples/address_test.go TestSystemSCsAddressesAndSpecialAddresses this address?

Reply:
done